ksql-engine/src/main/java/io/confluent/ksql/util/QueryLoggerUtil.java

```diff
@@ -34,4 +35,14 @@ public static String queryLoggerName(
             .addAll(Arrays.asList(subHierarchy))
             .build());
   }
+
+  public static String queryLoggerName(final QueryContext queryContext) {
```

Comment:
Now that we have this method, do we still need the other `queryLoggerName()` method in this file? Looks like it's unused.